client/post-editor/post-editor.jsx

```diff
@@ -231,9 +240,15 @@ export const PostEditor = React.createClass( {
 			<div className={ classes }>
 				<QueryPreferences />
 				<EditorConfirmationSidebar
-					hideSidebar={ this.hideConfirmationSidebar }
-					isActive={ this.state.showConfirmationSidebar }
+					closeOverlay={ this.closeConfirmationSidebarOverlay }
+					closeSidebar={ this.closeConfirmationSidebar }
+					isOverlayActive={ this.state.showConfirmationSidebarOverlay }
+					isSidebarActive={ this.state.showConfirmationSidebar }
```

Comment:
I'm don't see the need for passing overlay and sidebar close/active separately -- they seem like they are always done in a pair. I don't think the editor should know anything about the overlay.